services/expert-advice.html

```diff
@@ -5,36 +5,36 @@
 title: Expert Advice
 ---
 
-<div class="container content">
-    <div class="expert-section">
+<div class="container content" id="expert-advice">
+    <div class="services-section">
 	<img src="/assets/img/custom/services/expert_advice_icon_orange.png" />
-	<h2 class="expert-header margin-bottom-40">Expert Advice</h2>
+	<h2 class="services-header margin-bottom-40">Expert Advice</h2>
 	<p><strong>You can draw from our expertise to fast track your initiatives.</strong></p>
 	<p>You can draw on our experience to help you; start a new project, instil XP practices or Devops culture into your existing teams, plan your architectural evolution or adopt a new technology.</p>
```

Comment:
We can help you start a new project, instil XP practices or Devops culture into your existing teams, plan your architectural evolution or adopt a new technology.